packages/base-styles/README.md

```diff
@@ -0,0 +1,37 @@
+# Base styles
+
+Base scss utilities and variables for WordPress.
```

Comment:
```suggestion
Base SCSS utilities and variables for WordPress.
```
See: https://en.wikipedia.org/wiki/Sass_(stylesheet_language)

Reply:
Can we update also in all other places `scss` -> `SCSS`?

Reply:
Done https://github.com/WordPress/gutenberg/pull/17883/commits/071883cc5f98117dba99f0a29adeb782d22b162b